src/wfs/src/main/java/org/geoserver/wfs/GetFeature.java

```diff
@@ -563,9 +563,10 @@ public FeatureCollectionResponse run(GetFeatureRequest request)
                     isNumberMatchedSkipped = meta.getSkipNumberMatched()
                             && !request.isResultTypeHits();
                 if (!isNumberMatchedSkipped) {
+                		// Fix for GEOS-7233: totalFeatures for WFS 2.0 request returns wrong count.
```

Comment:
Please remove all these comments, we cannot have the source code riddled with "fix for blah blah" comments